docs/samples/contact-center/index.html

```diff
@@ -112,6 +112,7 @@ <h2 class="collapsible">
                   <input id="dialNumber" name="dialNumber" placeholder="Dial Number" value="" type="text">
                   <button id="loginAgent" disabled class="btn btn-primary my-3" onclick="doAgentLogin()">Login With
                     Selected Team</button>
+                  <button id="logoutAgent" style="display: none;" class="btn btn-primary my-3 ml-2" onclick="logoutAgent()">Logout Agent</button>
```

Comment:
Prefer using the already present 'hidden' class instead of display: none

Reply:
Done